.changeset/shiny-donkeys-matter.md

```diff
@@ -0,0 +1,6 @@
+---
+"@fuel-wallet/sdk": minor
+"@fuel-wallet/types": minor
+---
+
+feat(app): export vault secret phrase
```

Comment:
The changesets are only used for control SDK and types. When adding a screen, you can just add a `pnpm changeset add --empty`

Reply:
Perfect!